README.md

```diff
@@ -106,44 +112,46 @@ Other shortcuts:
 1. Click the "Finalize OTR" text in the green footer.
-1. Finally click "Generate OTR" and for the source destination it needs to go in the `mods` folder of your SoH installation.
-    - This `mods` folder should be a sibling of your `oot.otr` file.
+1. Click "Generate OTR" and, using the file selection screen, save the file to the `mods` folder of your SoH installation.
+    - This `mods` folder should be in the same folder as your `oot.otr` file.
 
-Assuming you have done everything correctly, boot up SoH and open up the SFX Editor (In the Enhancements dropdown). You should now be able to swap out any of the in game sequences/fanfares for the sequences added in your newly generated OTR file. If you have any trouble with this process please reach out in the support section of the Discord
+Assuming you have done everything correctly, boot up SoH and select the SFX Editor from the enhancements dropdown menu. You should now be able to swap out any of the in game sequences/fanfares for the sequences added in your newly generated OTR file. If you have any trouble with this process, please reach out in the support section of the Discord.
 
 ## Building The Ship of Harkinian
 
-Refer to the [building instructions](BUILDING.md) to compile SoH.
+If you want to manually compile SoH, please consult the [building instructions](BUILDING.md).
 
 ## Nightly Builds
+If you want to playtest a nightly build, you can find them at the links below. However, keep in mind that these are for playtesting only, and you will likely encounter bugs.  Rarely, these builds will crash.
+
 Nightly builds of Ship of Harkinian are available here: [Windows](https://nightly.link/HarbourMasters/Shipwright/workflows/generate-builds/develop/soh-windows.zip), [macOS](https://nightly.link/HarbourMasters/Shipwright/workflows/generate-builds/develop/soh-mac.zip), [Linux (compatibility*)](https://nightly.link/HarbourMasters/Shipwright/workflows/generate-builds/develop/soh-linux-compatiblity.zip), [Linux (performance*)](https://nightly.link/HarbourMasters/Shipwright/workflows/generate-builds/develop/soh-linux-performance.zip), [Switch](https://nightly.link/HarbourMasters/Shipwright/workflows/generate-builds/develop/soh-switch.zip), [Wii U](https://nightly.link/HarbourMasters/Shipwright/workflows/generate-builds/develop/soh-wiiu.zip)
 
 _*compatibility: compatible with most Linux distributions, but may not be as performant as the perf build._\
-_*performance: requires glibc 2.35 or newer, but will be more performant than the compat build._
+_*performance: requires `glibc 2.35` or newer, but will be more performant than the compat build._
 
 ## Take The Survey
-Want to use cartridge readers in tandem with the OTRGui?
+Want to use cartridge readers in tandem with OTRGui?
 Take [this survey](https://retroarchopenhardware.com/survey.php) to increase chances of this becoming reality.
 
 ## The Harbour Masters Are...
```

Comment:
@Kenix3 i feel like we should do something about everything below this. I feel like having a link to https://github.com/HarbourMasters/Shipwright/contributors and then adding a `CREDITS.md` or something might be better.

> The Ship of Harkinian is possible because of the hard work by [these wonderful contributors](https://github.com/HarbourMasters/Shipwright/contributors), as well as [many others](CREDITS.md)!

Reply:
also it'd be cool to get a "powered by libultraship" image

Reply:
This was the one section I didn't really want to touch because I felt like it wasn't my place. Let me know what you guys decide on!